wagtail/admin/wagtail_hooks.py

```diff
@@ -1207,3 +1207,11 @@ class WorkflowTaskAdminURLFinder(ModelAdminURLFinder):
     UnpublishBulkAction,
 ]:
     hooks.register("register_bulk_action", action_class)
+
+
+@hooks.register("insert_global_admin_js")
+def global_admin_js():
+    """
+    Adds the SEO character count JavaScript globally in the admin interface.
+    """
+    return '<script defer src="/static/wagtailadmin/js/seo_char_count.js"></script>'
```

Comment:
We don't use this hook for JS files that are in Wagtail itself. Instead, we just put it in the base admin template directly. However, we're also moving away from including individual files to something that would be easier to work with things like JS dynamic imports and importmaps.